### PR TITLE
Decentral Myvlog

### PR DESCRIPTION
**Pull request by DecentralTech**
Myvlog is a free app that automatically jump-cut edits your videos without the annoying editing.

### CODE REVIEW

**Comment by dineshshenoy** [comment]
Thanks for submitting an application.  Our team will validate for completeness and then assign reviewers from the competitions review panel.

**Comment by jasonleyser** [comment]
Hi @DecentralTech, thanks for your submission. I don't see your registration at [https://slingshot.filecoin.io/register-now](https://slingshot.filecoin.io/register-now). We require you to register each project before we can accept this submission. Thanks!

**Comment by DecentralTech** [comment]
@jasonleyser , I have finished the registration for this application, please check the inbox. Thanks in advance~~

File: participants/Decentral Myvlog.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+(0) Project Name and Team Slack Handles
+project's name:Decentral Myvlog
+Filecoin Slack: myvlog@decentral.cc
+
+(1) Brief Project Description
+Free and amazing app to record your vlog!
+
+(2) Link to Project UI
+Please include a link to your project application or UI.
+https://github.com/DecentralTech/myui
```
Comment (GregTheGreek) [inline participants/Decentral Myvlog.md:11]: Is there a live website for this, or just a plan to have a ui? The current UI seems to only contain images.

**Comment by jasonleyser** [comment]
> @jasonleyser , I have finished the registration for this application, please check the inbox. Thanks in advance~~

@DecentralTech, I received all of your registration submissions. Thanks!

File: participants/Decentral Myvlog.md (hunk 1 above)
Comment (OneilYang) [reply participants/Decentral Myvlog.md:11]: @GregTheGreek , ok, we're preparing more UI pages and features for the app, you will see it very soon...

**Comment by DecentralTech** [comment]
@GregTheGreek , ok, we're preparing more UI pages and features for the app, you will see it very soon...

File: participants/Decentral Myvlog.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+
+(0) Project Name and Team Slack Handles
+project's name:Decentral Myvlog
+Filecoin Slack: myvlog@decentral.cc
```
Comment (cwaring) [inline participants/Decentral Myvlog.md:4]: Is `@Decentral Tech` your username on slack?

**Comment by DecentralTech** [comment]
@GregTheGreek , the UI have been updated after some works, please see here: https://github.com/DecentralTech/myui/tree/master/ui/myvlog

File: participants/Decentral Myvlog.md (hunk 2 above)
Comment (DecentralTech) [reply participants/Decentral Myvlog.md:4]: Hi, @cwaring , the name on slack is @Decentral Myvlog, same to the project name, Thanks~~

Comment (cwaring) [inline participants/Decentral Myvlog.md:4]:
```suggestion
Filecoin Slack: myvlog@decentral.cc - @Decentral Myvlog

```

**Review by cwaring — APPROVED** [review]

**Pull request merged**